sdk/src/client/secret/types.rs

```diff
@@ -103,7 +103,7 @@ pub struct LedgerNanoStatus {
     /// Ledger is available and ready to be used.
     pub(crate) connected: bool,
     /// Ledger is connected and locked.
```

Comment:
Probably good to update the comments that it's only Some() for Shimmer and IOTA app

Reply:
Wanted to do this and forgot, thanks

Reply:
Done